bases/br_anp_precos_combustiveis/microdados/table_config.yaml

```diff
@@ -1,201 +1,346 @@
-source_bucket_name: basedosdados-dev
-project_id_staging: basedosdados-dev
-project_id_prod: basedosdados-dev
-table_id: microdados # AUTO GENERATED
-dataset_id: br_anp_precos_combustiveis # AUTO GENERATED
 
-url_ckan: https://basedosdados.org/dataset/br-anp-precos-combustiveis # AUTO GENERATED
-url_github: https://github.com/basedosdados/mais/tree/master/bases/br_anp_precos_combustiveis # AUTO GENERATED
+# Igual ao dataset.name mas como lower case.
+# Exemplos: br_ibge_populacao, br_inep_censo_escolar
+dataset_id: br_anp_precos_combustiveis
 
-version: v1.0 # REQUIRED
+table_id: microdados
 
-last_updated: 2021-10-18 # AUTO GENERATED
+# Título da tabela.
+title: Preços de Combustíveis - Dados históricos
 
 # Descreva a tabela. Essas são as primeiras frases que um usuário vai ver.
 # Você não precisa ser muito conciso. Sinta-se a vontade para dar exemplos de
 # como usar os dados.
-# Se souber, liste também aplicações: pesquisa, apps, etc. que usem os dados.
-description: |  # REQUIRED
-        Série Histórica de Preços de Combustíveis - a saber, gasolina, etanol, diesel s10, gasolina aditivada, glp, diesel e gnv - com base na pesquisa de preços da Agência Nacional do Petróleo, Gás Natural e Biocombustíveis realizada a partir da primeira semana de 2004 até os dias de hoje.
+# Se souber, liste também aplicações: pesquisa, apps, etc. que usem os dados.,
+description: Série Histórica de Preços de Combustíveis - a saber, gasolina, gasolina aditivada, etanol, diesel s10, diesel, gnv e  glp - com base na pesquisa de preços da Agência Nacional do Petróleo, Gás Natural e Biocombustíveis realizada a partir da primeira semana de 2004 até os dias de hoje.
+
+# As máximas unidades espaciais que a tabela cobre.
+# Exemplo:
+#    - sa.br
+#  
+#    - sa.br.sp
+#  
+#    - world
+spatial_coverage:
+    - sa.br
+
+# Anos cobertos pela tabela.
+# Exemplos:
+#     - 1995(1)2019
+# Caso a cobertura não seja contínua:
+#     - 2002(2)2010
+#     - 2016
+#     - 2020
+temporal_coverage:
+    - 2004(1)2022
```

Comment:
Acho que seria 2004(1)2023

Reply:
é uma boa indicar o mês de atualização: 
`ex. 2004-01(1)2023-03`

ps: acho que o ideal seria a nomenclatura pra semana

Reply:
Então, eu to subindo os dados 2004(1)2022. Os dados de 2023 entrarão pela pipeline atualizados mensalmente.